fsspec/compression.py

```diff
@@ -70,7 +69,22 @@ def unzip(infile, mode="rb", filename=None, **kwargs):
 
 register_compression("zip", unzip, "zip")
 register_compression("bz2", BZ2File, "bz2")
-register_compression("gzip", lambda f, **kwargs: GzipFile(fileobj=f, **kwargs), "gz")
+
+try:
```

Comment:
could you please mark this block as not covered, since we won't install isal on CI?

Reply:
@martindurant, what do you mean exactly? just a comment ?

Reply:
Add `  # pragma: no cover` to the end of the line with `try:` will indicate to `coverage` that we understand that this block is not used in the CI.

Reply:
Ok thanks. done!